data/model/modelutil.py

```diff
@@ -69,24 +77,32 @@ def pagination_start(page_token=None):
     return None
 
 
-def paginate_query(query, limit=50, sort_field_name=None, page_number=None):
+def paginate_query(query, limit=50, sort_field_name=None, page_number=None, offset_val=0):
     """
     Executes the given query and returns a page's worth of results, as well as the page token for
     the next page (if any).
     """
     results = list(query)
     page_token = None
+    offset_val = 0
```

Comment:
This is already defined as a default parameter

Reply:
Ah, removed.